### ISSUE TITLE
DocFx does not support  /// <exclude />

### ISSUE THREAD

**Issue by ThomCBlackwellBSS** [dotnet/docfx#5796]
**Operation System**:Windows

**DocFX Version Used**: Most recent

**Template used**: Default

**Steps to Reproduce**:

1. added triple slash <exclude /> above a property "/
    /// \<exclude />
        public string SomePropertyIDoNotWantDocumented { get; set; } = "";
**Expected Behavior**:
Property documentation would not be generated

**Actual Behavior**:
Property documentation is generated

**Comment by thomb1** [comment]
Note the above window does not allow me to put the triple slash comment. It is the standard Exclude tag that GhostDoc adds when you select Exclude This from Documentation.

**Comment by balajirenganathan** [comment]
I'm also facing the same issue, excluded public API's in the sources were generated in the output document. Any workaround and idea to resolve this?

/// <exclude />

**Comment by thomb1** [comment]
Just saw your post balajirenganathan.

I agree DocFx should support Exclude. This is unnecessary extra and arcane work to support a common feature need (easily filter out items from generated end user documentation. 
-----------------------------------
 Here is what I ended up doing: 
I added an custom attribute
       [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]

I set Filter in my docfx.json file
inside
 "build": {
... 
I set
    "filter": "filter.yml",

Here is the part of my filter.yml that checks for that attribute.

apiRules:
- exclude:
      uidRegex: ^System\.Object
      type: Type
- exclude:
    hasAttribute:
      uid: System.ComponentModel.EditorBrowsable
